### PR TITLE
4342 // periodically run public studios tests

### PR DESCRIPTION
**Pull request by Dinika**
Fixes #

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added necessary unit and integration tests.
- [ ] I have added screenshots (if applicable), in the comment section.

### CODE REVIEW

File: .github/workflows/monitor-studios.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: Test Public Studios
+on:
+  schedule:
+    cron: '0 0 12 * * ?' # Fire at 12:00 p.m. (noon) every day
+
+jobs:
+  test-and-build:
```
Comment (bilalesi) [inline .github/workflows/monitor-studios.yml:7]: small thing: may `e2e-studio-cron-test ` or anything you prefer

File: .github/workflows/monitor-studios.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    runs-on: it
```
Comment (bilalesi) [inline .github/workflows/monitor-studios.yml:8]: you should remove the it to run it in the free runner

Comment (bilalesi) [reply .github/workflows/monitor-studios.yml:8]: you can use `runs-on: ubuntu-latest`

Comment (Dinika) [reply .github/workflows/monitor-studios.yml]: Thanks! Going to try that.

File: .github/workflows/monitor-studios.yml (hunk 1 above)
Comment (Dinika) [reply .github/workflows/monitor-studios.yml]: Good point. Changed it now.

**Review by bilalesi — APPROVED** [review]

**Pull request merged**